### PR TITLE
Change how the incompatible-browser script loads

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

This will help us eliminate some CSP errors that stem from having a script with a `src` attribute in our liquid templates.

This is modeled after static-page-widgets:

https://github.com/department-of-veterans-affairs/vets-website/blob/d36d1c27f62328143d188cee7dbcf26488eda603/src/site/includes/footer.html#L10-L16

https://github.com/department-of-veterans-affairs/vets-website/blob/d36d1c27f62328143d188cee7dbcf26488eda603/src/site/assets/js/static-page-widgets.js#L48-L50

image

## Screenshots

The inlined script:

image

I'm not entirely sure that we need the whole `module` thing, but I'm following a pre-existing pattern with this.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/landing-pages/dev-template.ejs [hunk 1]
```diff
@@ -121,8 +121,13 @@
 
     <!-- TODO: Add breadcrumbs -->
 
-    <script async src="/js/incompatible-browser.js"></script>
-
+    <script async type="text/javascript">
```
Comment (cvalarida) [inline src/platform/landing-pages/dev-template.ejs:124]: Do we need `async` without a `src`?

File: src/site/includes/header.html [hunk 2]
```diff
@@ -89,4 +89,10 @@
     {% include "src/site/includes/homepage-banner.liquid" with banner = homepage_banner %}
   {% endif %}
 
-  <script async src="/js/incompatible-browser.js"></script>
+  <script async type="text/javascript">
```
Comment (cvalarida) [inline src/site/includes/header.html:92]: Same here. (This one is the one that counts. :stuck_out_tongue: )

File: src/platform/landing-pages/dev-template.ejs (hunk 1 above)
Comment (bkjohnson) [reply src/platform/landing-pages/dev-template.ejs:124]: Nope!

**Review by cvalarida — APPROVED** [review]
I see what you're saying about the `module` bit. Not sure we do need it. :man_shrugging: 

This works for small files like this, but I'm concerned about the performance implication for JS files in `js/` that are larger than this. We may need to come up with another approach.

**Pull request merged**